Makefile

```diff
@@ -21,6 +21,9 @@ endif
 TEST_ARTIFACTS_DIR := integration-test/test-artifacts
 BUNDLE_ARCHIVES_DIR := $(TEST_ARTIFACTS_DIR)/archives
 
+BENCH_ARTIFACTS_DIR := runtime/test-artifacts
+BENCH_BUNDLE_DIR := $(BENCH_ARTIFACTS_DIR)/archives
```

Comment:
Avoided duplication of rootfs archives. Also test and benchmark outputs will be under a common directory called "output" under the git repo root. Updated relevant integration-test code.